### ISSUE TITLE
Possible incompatibility with fluffy's work tab?

### ISSUE THREAD

**Issue by Amnesiac9** [UnlimitedHugs/RimworldMapReroll#4]
Started getting this upon adding only this mod and starting a new game

https://i.imgur.com/duiRnEC.png

**Comment by Amnesiac9** [comment]
I also get this when trying to save the game https://i.imgur.com/sq0vKeF.png

**Comment by UnlimitedHugs** [comment]
I'll investigate.

**Comment by Amnesiac9** [comment]
Thanks! If it turns out its not this mod and rather an incompatibility, I do hope its fixable, but I understand if you have more important things to do. Love the mods regardless. 

Here's a mod list of what I had enabled before adding Map Reroll:


<modNames>
			<li>Core</li>
			<li>HugsLib</li>
			<li>EdB Prepare Carefully</li>
			<li>Allow Tool</li>
			<li>Animal Tab</li>
			<li>Medical Tab</li>
			<li>Relations Tab</li>
			<li>Work Tab</li>
			<li>Blueprints</li>
			<li>Research Tree</li>
			<li>Colony Manager</li>
			<li>Dubs Mint Menus</li>
			<li>Fluffy Breakdowns</li>
			<li>ExtendedInspectData [B19]</li>
			<li>Hand Me That Brick Lite</li>
			<li>Pick Up And Haul</li>
			<li>Dismiss Trader</li>
			<li>P-Music</li>
			<li>VGP Vegetable Garden</li>
			<li>VGP_CoffeeTeaDrugs</li>
			<li>VGP Garden Medicine</li>
			<li>Heat Map</li>
			<li>RT Fuse</li>
			<li>Wandering Caravans [v2.3.5]</li>
			<li>[KV] Show Hair With Hats or Hide All Hats - B19</li>
			<li>ED-Embrasures</li>
			<li>[B19] Wild Animal Sex V3.2</li>
			<li>VGP Garden Tools</li>
			<li>Room Food</li>
			<li>[KV] RimFridge - B19</li>
			<li>PowerSwitch</li>
			<li>Animals Logic</li>
			<li>Stack XXL</li>
		</modNames>

**Comment by UnlimitedHugs** [comment]
Looks like Work Tab is not to blame- at least, it's not giving me any issues.
It must be an incompatibility with another one of your mods- if you can trim down that list to less than 10 items and still have errors, I'll take a look.

**Comment by Amnesiac9** [comment]
Alright so I did a bunch of testing, looks like its related to Prepare Carefully and Work Tab.

I first started a game, removed each mod, then loaded the game, and got the errors each time, except when I removed the following mods:

MapReroll
OR
WorkTab

But since you said they were not the issue, I tried starting a new game with the original mod list, plus Map Reroll, but instead I removed **Prepare Carefully.** This fixed the issue, no errors upon saving and loading the game.

I then started a new game with ONLY prepare carefully, and no errors. 
Then I added just WorkTab, and the errors showed up!
Then I removed WorkTab, and added MapReroll, and No Errors. 

So the issue is with Prepare Carefully and Fluffies work Tab! Only shows up when choosing to "prepare carefully" when making the pawns.

Which is very strange, because when I was playing earlier I didn't have any errors until I added Map Reroll, and that was the only mod I added, and when I removed it, the errors stopped when I loaded the save so I assumed it was this mod, but perhaps my earlier game I did not add worktab until after I started the game, so the errors never happened. 

So sorry to bother you, I will send the logs to the Prepare Carefully and WorkTab authors, and hopefully they can find out what's going on.
